source/guides/how-to-run-a-design-system-team-ceremony-or-workshop.html.md

```diff
@@ -23,123 +23,64 @@ There are some principles that are important to remember when planning and facil
+### Cycle planning 
 
-It’s important to ensure each goal has an assignee(s) who will work on the goal. Do this during the session.
+Cycle planning occurs every reflection week.
 
-At the end of the session, ensure all goals are in the “Sprint backlog” ready to be worked on and assigned to the correct “Iteration” on the board (eg Q2 22/23 Sprint 2).
+In these sessions, the team reviews the briefs prioritised for the next cycle. Because of this, it’s important that the whole squad attends. These sessions are also recorded to ensure anyone on leave can later re-watch.
 
-### Backlog gardening
+The briefs are prepared beforehand by the team leads and drafted into GitHub issues. [Epic leads](https://govuk-design-system-team-docs.netlify.app/guides/how-to-be-an-epic-lead/) are identified by the team leads and informed of their role before cycle planning. Squads for each brief are also currently determined beforehand, but in future we'd like to move to a self-select model during planning.
 
-Backlog gardening occurs for the Frontend squad within the team every other Tuesday.
+In the session, the facilitator will recap the briefs from the previous cycle and introduce the upcoming briefs and squads. They will also highlight anything of note happening in the next cycle (eg significant leave or events which may impact delivery). We use [the same slide deck](https://docs.google.com/presentation/d/1MxoMVQe7DgpdQVXLqRZVU6TYkqBZyH7cAS75W-PMVTE/edit#slide=id.g292361900d8_0_4) for each planning session but just update the details each time.
 
-The purpose of this session is to make sure that the backlog has all the right work in it, that the work is well described, and that it's ordered well. It’s a fancy way of saying backlog refinement.
-
-Think ahead over the next 4 weeks and consider which epics – or tasks in epics – the team is likely to start working on soon. Ensure that the outcomes are clear and any dependencies are noted: the goal is to make sure that an issue is well described and that someone could pick it up independently. For example, if it’s a spike that should result in a team workshop or learning session, make sure that’s a part of the issue or epic.
-
-This is also an opportunity to think about any pre-work or knowledge sharing that needs to happen. For example, if a piece of delivery work needs scoping, you’ll probably want to arrange a team workshop to help, like MoSCoW prioritisation or getting feedback from service teams.
-
-Where a piece of work can help someone meet their professional development objectives, highlight that they might want to work on it – or assign it to them and let others on the team know they’ll be tackling it.
-
-Note: This ceremony is fairly organic and has suited the team OK so far, but we have identified a need to increase collaboration and pairing on the team. So it should probably be amended / adapted as we tweak our processes to meet the needs of the team.
-
-### Mid-sprint review
```

Comment:
Do you want to add a couple of lines about Epic checkin to replace mid sprint review?